src/qumin/representations/utils.py

```diff
@@ -101,6 +117,18 @@ def aggregator(s):
             log.info("Dropping the columns named 'variants'")
             paradigms.drop("variants", axis=1, inplace=True)
 
+        if cells is not None:
+            par_cols = paradigms.columns
+            for checkname in cells:
+                if checkname not in par_cols:
+                    raise ValueError(f"It's impossible to keep the \
+                                    column {checkname}, since it doesn't exist.")
+            cells.append('lexeme')
+            todrop = list(set(par_cols)-set(cells))
+            if len(todrop) > 0:
+                log.info('Dropping unnecessary columns : '+", ".join(sorted(todrop)))
```

Comment:
Homogeneize message with the above,  eg. "Dropping columns with following cell headers:"